src/core/tests/utils.test.tsx

```diff
@@ -140,9 +145,16 @@ describe('core/utils', () => {
       expect(replaceEqualDeep(null, null)).toBe(null)
       expect(replaceEqualDeep(undefined, undefined)).toBe(undefined)
     })
+
+    it('should return the previous value when the next value is an equal Date', () => {
+      const a = new Date('December 17, 1995 03:24:00')
```

Comment:
nothing special about this date BTW. I stole it from https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/Date